bin/run-browser-test-env

```diff
@@ -75,46 +75,59 @@ EOF
   done
 }
 
+#######################################
+# Chooses the dev or ci versions of broswertest
+# Arguments:
+#   1: if true, use developer mode
+#   2: the rest of the arguments for compose
+#######################################
+function compose_browser_test() {
+  local use_dev="$1"
+  shift 1
+  if [ "$use_dev" = true ]; then
```

Comment:
suggestion (blocking): double brackets to conform to style guide

Reply:
ah actually can use the `truth::` functions: `truth::is_enabled use_dev` after making other suggested changes